server/constants/routes.go

```diff
@@ -2,18 +2,18 @@ package constants
 
 const (
 	// Plugin API Routes
-	APIPrefix                    = "/api/v1"
-	WildRoute                    = "{anything:.*}"
-	PathOAuthConnect             = "/oauth/connect"
-	PathOAuthCallback            = "/oauth/complete"
-	PathGetAllLinkedProjects     = "/project/link"
-	PathUnlinkProject            = "/project/unlink"
-	PathUser                     = "/user"
-	PathCreateTasks              = "/tasks"
-	PathLinkProject              = "/link"
-	PathCreateSubscriptions      = "/subscriptions"
-	PathNotificationSubscription = "/subscriptions"
-	PathGetSubscriptions         = "/subscriptions"
+	APIPrefix                     = "/api/v1"
+	WildRoute                     = "{anything:.*}"
+	PathOAuthConnect              = "/oauth/connect"
+	PathOAuthCallback             = "/oauth/complete"
+	PathGetAllLinkedProjects      = "/project/link"
+	PathUnlinkProject             = "/project/unlink"
+	PathUser                      = "/user"
+	PathCreateTasks               = "/tasks"
+	PathLinkProject               = "/link"
+	PathGetSubscriptions          = "/subscriptions"
```

Comment:
here there is no need to create 2 constants with similar values just have one constants and use it for all crud operations
```suggestion
	PathSubscriptions          = "/subscriptions"
```